apps/etl/transform/sources/base.py

```diff
@@ -15,13 +15,12 @@
     "emdat-impacts": PyStacLoadData.ItemType.IMPACT,
 }
 
-# XXX: Local file is used
-GEOCODER = GAULGeocoder(gpkg_path="/code/gaul2014_2015.gpkg")
-
 
 @shared_task
 def transform_data(source, transformer, data_source, extraction_id, data):
     logger.info(f"Transformation started for {source} data")
+
+    geocoder = GAULGeocoder(gpkg_path="/code/gaul2014_2015.gpkg")
```

Comment:
@Rup-Narayan-Rajbanshi Use env variable to control whether to use the local file or the service url. What do you think?

Reply:
For these cases we have this script https://github.com/IFRCGo/montandon-etl/blob/develop/apps/etl/management/commands/local_cache_pull.py and settings.LOCAL_CACHE_DATA_DIR variable.

Let's use that for stuffs we have to download and use.